u_parrots.go

```diff
@@ -208,6 +209,7 @@ func (uconn *UConn) applyPresetByID(id ClientHelloID) (err error) {
 }
 
 // ApplyPreset should only be used in conjunction with HelloCustom to apply custom specs.
+// Separate ClientHelloSpec for each UConn is advised, to avoid state sharing.
```

Comment:
Is this because `ApplyPreset` modifies `p`?  If so, that could be made explicit here.

Reply:
It doesn't modify state of `p` anymore (at this time).